### PR TITLE
Sign up feature added

### PR DESCRIPTION
**Pull request by ARRiera96**

### CODE REVIEW

File: server/app/configure/authentication/index.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ module.exports = function (app, db) {
         req.logout();
         res.status(200).end();
     });
-
+    console.log("I'm here at auth");
```
Comment (jdhang) [inline server/app/configure/authentication/index.js:70]: Delete this @ARRiera96 👍

**Comment by jdhang** [comment]
Looks pretty good to me

File: browser/js/fsa/fsa-pre-built.js [hunk 2]
```diff
@@ -101,6 +101,14 @@
             });
         };
 
+        this.signup= function(credentials){
+            return $http.post('/signup', credentials)
+            .then(onSuccessfulLogin)
+            .catch(function(){
+                return $q.reject({ message: 'Invalid login credentials.' });
```
Comment (elizabethkim828) [inline browser/js/fsa/fsa-pre-built.js:108]: invalid signup credentials

File: browser/js/signup/signup.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	$scope.signup = {};
+    $scope.error = null;
+
+  $scope.sendCredentials = function (creds) {
+    AuthService.signup(creds)
+    .then(function () {
+      $state.go('home');
+    })
+    .catch(function(){
+    	$scope.error= 'Invalid login credentials.';
```
Comment (elizabethkim828) [inline browser/js/signup/signup.js:24]: invalid signup credentials

**Pull request merged**